Build error on target armv7-linux-androideabi with rustc 1.73

Hi 

Did someone encounter these errors? How to fix it? thanks

error[E0658]: use of unstable library feature 'io_error_other'
  --> /home/bojan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compression-codecs-0.4.32/src/flate/decoder.rs:54:37
   |
54 |             Status::BufError => Err(io::Error::other("unexpected BufError")),
   |                                     ^^^^^^^^^^^^^^^^
   |
   = note: see issue #91946 <https://github.com/rust-lang/rust/issues/91946> for more information

error[E0658]: use of unstable library feature 'io_error_other'
  --> /home/bojan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compression-codecs-0.4.32/src/flate/decoder.rs:88:37
   |
88 |             Status::BufError => Err(io::Error::other("unexpected BufError")),
   |                                     ^^^^^^^^^^^^^^^^
   |
   = note: see issue #91946 <https://github.com/rust-lang/rust/issues/91946> for more information

error[E0658]: use of unstable library feature 'io_error_other'
  --> /home/bojan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compression-codecs-0.4.32/src/flate/encoder.rs:55:37
   |
55 |             Status::BufError => Err(io::Error::other("unexpected BufError")),
   |                                     ^^^^^^^^^^^^^^^^
   |
   = note: see issue #91946 <https://github.com/rust-lang/rust/issues/91946> for more information

error[E0658]: use of unstable library feature 'io_error_other'
  --> /home/bojan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compression-codecs-0.4.32/src/flate/encoder.rs:98:37
   |
98 |             Status::BufError => Err(io::Error::other("unexpected BufError")),
   |                                     ^^^^^^^^^^^^^^^^
   |
   = note: see issue #91946 <https://github.com/rust-lang/rust/issues/91946> for more information

error[E0658]: use of unstable library feature 'io_error_other'
  --> /home/bojan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compression-codecs-0.4.32/src/gzip/encoder.rs:75:32
   |
75 |                     return Err(io::Error::other("encode after complete"));
   |                                ^^^^^^^^^^^^^^^^
   |
   = note: see issue #91946 <https://github.com/rust-lang/rust/issues/91946> for more information

error[E0658]: use of unstable library feature 'io_error_other'
   --> /home/bojan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compression-codecs-0.4.32/src/gzip/header.rs:156:32
    |
156 |                     return Err(io::Error::other("parser used after done"));
    |                                ^^^^^^^^^^^^^^^^
    |
    = note: see issue #91946 <https://github.com/rust-lang/rust/issues/91946> for more information

For more information about this error, try `rustc --explain E0658`.

--- a/crates/compression-codecs/Cargo.toml
+++ b/crates/compression-codecs/Cargo.toml
@@ -10,7 +10,7 @@ edition.workspace = true
 license.workspace = true
 readme = "../../README.md"
 repository.workspace = true
-rust-version = "1.74"
+rust-version = "1.83"
 
 [package.metadata.docs.rs]
 all-features = true

